src/tcgwars/logic/impl/pokemod/PokemodBaseSet.groovy

```diff
@@ -1619,8 +1619,7 @@ public enum PokemodBaseSet implements LogicCardInfo {
           def stage = pkmn.size()>1 ? pkmn.select("Choose stage to devolve to").first() : pkmn.first()
```

Comment:
Same as the other comment, Also, don't think lv.xs are planned for pokemod (yet) but might be worth adding checks for it here (and def would need to be added in BaseSetNG for Unlimited)